### ISSUE TITLE
Hotkeys not working in 1.6.2.0

### ISSUE THREAD

**Issue by xenolightning** [xenolightning/AudioSwitcher_v1#107]
Hotkeys doesn't work for me

Speaking of hotkeys, I can't seem to get them to work. I've tried several different configurations and I also tried 1.6.1.9 (as I had the crash-on-start issue with 1.6.1.8) and it didn't work with that one either. I've tried experimenting with a several settings but nothing seems to make hotkeys work for me unfortunately.

**Comment by Timmyfox** [comment]
It seems like the hotkeys are working for me now with the 1.6.3.0 update, so whatever you did it may have fixed this!

**Comment by xenolightning** [comment]
Hah, awesome.

I found a pretty core bug that was preventing the App from closing correctly. Was noticeable by how long the exe took to close.

Fixing this may have helped clean up process of the hotkeys on close.

The other fix that was introduced was the new settings file, you may have noticed it's now a .json file. MUCH more stable when storing the settings, which could've helped out with this too.

tldr; No idea what fixed it, but glad it's working again :P


On another note, the "core bug" was discovered while developing Audio Switcher 2.0, which is coming along nicely.
